![20190922_164320](https://user-images.githubusercontent.com/22411298/71449590-c51eac00-2775-11ea-844f-b0a18ab220c9.jpg)
## Atanu Bhattacharjee,Ph.D (Statistics)
Assistant Professor Biostatistics
Centre for Cancer Epidemiology, Tata Memorial Centre, India

## Biography
I am working as Assistant Professor Biostatistics in the Section of Biostatistics, Centre for Cancer Epidemiology, Tata Memorial Centre, Mumbai.I am also an Ajunct Professor of Statistics at Department of Applied Mathematics, Indian Institute of Technology-Dhanbad. I completed by Ph.D in statistics under the supervision of Dr. Dilip C Nath.
My research interests include Bayesian inference, longitudinal data analysis, survival analysis, gene expression data analysis.

### Publication
## [2019]
1. Amit Joshi, Sameer Shrirangwar, Vanita Noronha, Nilesh Sable, Archi Agarwal, Palak Popat, Atanu Bhattacharjee, Kumar Prabhash.
 A tertiary care audit of using abiraterone acetate in patients of metastatic castrate-resistant prostate cancer.South Asian J
Cancer 2020;9:23-6.
2. Thiagarajan S, Dhar H, Bhattacharjee A, Fatehi KS, Shah SB, Chaukar D, Nair D, Deshmukh A, Prabhash K, Joshi A, Patil V, Noronha V, Laskar SG, Cruz AKD.Patterns of failure and outcomes in cT4 Oral squamous cell carcinoma (OSCC) undergoing upfront surgery in comparison to Neo-Adjuvant Chemotherapy (NACT) followed by surgery: A Matched Pair analysis.Oral Oncol. 2019 Nov 15;100:104455. doi: 10.1016/j.oraloncology.2019.104455.
3. Narasimhan M, Kannan S, Chawade A, Bhattacharjee A, Govekar R. Clinical biomarker discovery by SWATH-MS based label-free quantitative proteomics: impact of criteria for identification of differentiators and data normalization method.J Transl Med. 2019 May 31;17(1):184. doi: 10.1186/s12967-019-1937-9.
4. Patil V#1,2, Joshi A#1,2, Noronha V1,2, Agarwala V1,2, Chougule A1,2, Kanan S, Bhattacharjee A, Chandrasekharan A, Pande N, Simha V, Goud S, More S, Kumar R, Mahajan A, Janu A, Purandare N, Prabhash K.Randomized phase 3 open label study of quality of life of patients on Pemetrexed versus Erlotinib as maintenance therapy for advanced non squamous non EGFR mutated non small cell lung cancer.Oncotarget. 2019 Oct 29;10(59):6297-6307. doi: 10.18632/oncotarget.27214. eCollection 2019 Oct 29.
5. Singhavi HR, Chakrabarti S, Singh A, Mair M, Bhattacharjee A, Shetty R, Qayyumi B, Pai A, Joshi P, Nair D, Nair S, Chaturvedi P.Comparison of the 7th and 8th edition AJCC classification system in oral cavity squamous cell cancers.Int J Cancer. 2019 Oct 3. doi: 10.1002/ijc.32720. 
6. Vijay Patil, Vanita Noronha, Amit Joshi, Anuradha Chougule, Atanu Bhattacharjee, Alok Goel, Vikas Talreja, Nandini Menon, Anant Ramaswamy, Ashay Karpe, Nikhil Pande, Arun Chandrasekharan, Abhishek Mahajan, Amit Janu, Nilendu Purandare, Kumar Prabhash.
Quality-adjusted time without symptoms or toxicity (Q-TWiST) analysis of a Phase III randomized trial to compare the benefit of gefitinib versus pemetrexed/ carboplatin for epidermal growth factor receptor-mutated non-small cell lung cancer.Cancer Research,Statistics and Treatment.2019 2(1).21-27.
7. Patil VM, Noronha V, Joshi A, Agarwal J, Ghosh-Laskar S, Budrukkar A, Murthy V, Gupta T, Mahimkar M, Juvekar S, Arya S, Mahajan A, Agarwal A, Purandare N,Rangarajan V, Balaji A, Chaudhari SV, Banavali S, Kannan S, Bhattacharjee A, D'Cruz AK, Chaturvedi P, Pai PS, Chaukar D, Pantvaidya G, Nair D, Nair S, Deshmukh A,Thiagarajan S, Mathrudev V, Manjrekar A, Dhumal S, Maske K, Bhelekar AS, Nawale K, Chandrasekharan A, Pande N, Goel A, Talreja V, Simha V, Srinivas S, Swami R,Vallathol DH, Dsouza H, Shrirangwar S,Turkar S, Abraham G, Thanky AH,Patel U,Pandey MK,Prabhash K.A randomized phase 3 trial comparing nimotuzumab plus cisplatin chemoradiotherapy versus cisplatin chemoradiotherapy alone in locally advanced head and neck cancer.Cancer. 2019 Sep 15;125(18):3184-3197. doi: 10.1002/cncr.32179. Epub 2019 May 31.
8. Noronha V, Patil VM, Joshi A, Menon N, Chougule A, Mahajan A, Janu A, Purandare N, Kumar R, More S, Goud S, Kadam N, Daware N, Bhattacharjee A, Shah S, Yadav A,
Trivedi V, Behel V, Dutt A, Banavali SD, Prabhash K.Gefitinib Versus Gefitinib Plus Pemetrexed and Carboplatin Chemotherapy in EGFR-Mutated Lung Cancer.J Clin Oncol. 2019 Aug 14:JCO1901154. doi: 10.1200/JCO.19.01154.
9. Bhattacharjee A.Bayesian competing risks analysis without data stratification.Clinical Epidemiology and Global Health (In press) 2019.
10.Mukesh Kumar  Prashant Kr. Sonker  Agni Saroj  Aanchal Jain  Atanu Bhattacharjee  Rakesh Kr. Saroj.
Cancer Reports.Parametric survival analysis using R: Illustration with lung cancer data.2019. https://doi.org/10.1002/cnr2.1210
11. Bhattacharjee A.A joint longitudinal and survival model for dynamic treatment regimes in Presence of Competing Risk Analysis.
Clinical Epidemiology and Global Health.2019.7(3).337-341.
12. Bhattacharjee A,Budukh A,Dikshit  R.Prostate cancer survival estimates: An application with piecewise hazard function derivation.South Asian journal of cancer.2019.8(3).105.
13. A Bhattacharjee, GK Vishwakarma.Time-course data prediction for repeatedly measured gene expression.International Journal of Biomathematics.2019.12(4).
14. Vijay M Patil, Arun Chandrasekharan, Dilip Harindran Vallathol, Mridul Malhotra, Ram Abhinav, Priti Agarwal, Anu Rajpurohit, Raees Tonse, Atanu Bhattacharjee, Rakesh Jalali. Antiemetic prophylaxis with temozolomide: an audit from a tertiary care center.2019.Neuro-Oncology Practice.4(19).
15. Atanu Bhattacharjee, Gaurav Roy, Atul Budukh, Rajesh Dikshit, Vijay M Patil, Amit Joshi, Vanita Noronha, Kumar Prabash, Papai Roy A competing risk analysis of death patterns in male genitourinary cancer.(2019).Cancer Reports, e1174.


### Biostatistics Tutorial
1. Introduction to Biostatistics
[1546938553M1.pdf](https://github.com/atanustat/biostatonco/files/4000779/1546938553M1.pdf)
2. Survival Analysis
[Survival Analysis.pdf](https://github.com/atanustat/biostatonco/files/4000798/Survival.Analysis.pdf)
3. Parametric survival analysis
[parametric survival analysis.pdf](https://github.com/atanustat/biostatonco/files/4000800/parametric.survival.analysis.pdf)
4. Acturial estimator with R
[Acturial estimator with R.pdf](https://github.com/atanustat/biostatonco/files/4000801/Acturial.estimator.with.R.pdf)
5. Kaplan Meier Estimators-I
[Kaplan Meier Estimators-I.pdf](https://github.com/atanustat/biostatonco/files/4000802/Kaplan.Meier.Estimators-I.pdf)
6. Kaplan Meier Estimators-II
[Uploading Kaplan Meier Estimators-Ii.pdf…]()
7. Nelson aalen estimator
[nelson aalen estimator.pdf](https://github.com/atanustat/biostatonco/files/4001281/nelson.aalen.estimator.pdf)
8. Equality of survival
[equality of survival function.pdf](https://github.com/atanustat/biostatonco/files/4001282/equality.of.survival.function.pdf)
9. Mantel Haenszel Estimator
[Mantel-Haenszl Estimators.pdf](https://github.com/atanustat/biostatonco/files/4001284/Mantel-Haenszl.Estimators.pdf)
10. Cox PH Model
[Cox PH model.pdf](https://github.com/atanustat/biostatonco/files/4001285/Cox.PH.model.pdf)

11. Estimation and Inference for Cox’s [Estimation and Inference for Cox’s.pdf](https://github.com/atanustat/biostatonco/files/4001298/Estimation.and.Inference.for.Cox.s.pdf)
12. Diagnostics for Cox’s PH model [Diagnostics for Cox’s PH model.pdf](https://github.com/atanustat/biostatonco/files/4001299/Diagnostics.for.Cox.s.PH.model.pdf)
13. The Accelerated Failure Time [The Accelerated Failure Time.pdf](https://github.com/atanustat/biostatonco/files/4001300/The.Accelerated.Failure.Time.pdf)
14. Competing Risks Model [Competing Risks Model.pdf](https://github.com/atanustat/biostatonco/files/4001301/Competing.Risks.Model.pdf)
15. Clinical Trials [Clinical Trials.pdf](https://github.com/atanustat/biostatonco/files/4001302/Clinical.Trials.pdf)
16. Sample size determination
[sample size determination.pdf](https://github.com/atanustat/biostatonco/files/4001312/sample.size.determination.pdf)
17. Random Allocation
[random allocation.pdf](https://github.com/atanustat/biostatonco/files/4001314/random.allocation.pdf)
18. Sequential Clinical Trial-I
[sequential clinical trial.pdf](https://github.com/atanustat/biostatonco/files/4001315/sequential.clinical.trial.pdf)
19. Sequential Clinical Trial-II
[sequential clinical trial II.pdf](https://github.com/atanustat/biostatonco/files/4001316/sequential.clinical.trial.II.pdf)
20. Sequential Clinical Trial-III
[sequential clinical trial III.pdf](https://github.com/atanustat/biostatonco/files/4001318/sequential.clinical.trial.III.pdf)
21. Randomize play the winder rule
[Randomize play the winder rule.pdf](https://github.com/atanustat/biostatonco/files/4001329/Randomize.play.the.winder.rule.pdf)
22. The Play the Winnder Rule [The Play the Winnder Rule.pdf](https://github.com/atanustat/biostatonco/files/4001331/The.Play.the.Winnder.Rule.pdf)
23. Crossover Trial[Crossover trial.pdf](https://github.com/atanustat/biostatonco/files/4001332/Crossover.trial.pdf)
24. Introduction to Epidemiology[Introduction to Epidemiology.pdf](https://github.com/atanustat/biostatonco/files/4001333/Introduction.to.Epidemiology.pdf)
25. Disease Frequency[Disease Frequency-I.pdf](https://github.com/atanustat/biostatonco/files/4001335/Disease.Frequency-I.pdf)
26. Disease Frequency-II[Disease Frequency-II.pdf](https://github.com/atanustat/biostatonco/files/4001354/Disease.Frequency-II.pdf)
27. Disease Frequency-III[Disease Frequency-III.pdf](https://github.com/atanustat/biostatonco/files/4001358/Disease.Frequency-III.pdf)
28. Observational Study[Observational Study.pdf](https://github.com/atanustat/biostatonco/files/4001360/Observational.Study.pdf)
29. Analystical Study[Analytical Study.pdf](https://github.com/atanustat/biostatonco/files/4001353/Analytical.Study.pdf)
30. Case Control Study[Case Control Study.pdf](https://github.com/atanustat/biostatonco/files/4001352/Case.Control.Study.pdf)
31. Cohort Study-I [Cohort Study -I.pdf](https://github.com/atanustat/biostatonco/files/4001350/Cohort.Study.-I.pdf)
32. Cohort Study-II [Cohort Study-II.pdf](https://github.com/atanustat/biostatonco/files/4001349/Cohort.Study-II.pdf)
33.  Case Control Study -I[case Control Study -I.pdf](https://github.com/atanustat/biostatonco/files/4001346/case.Control.Study.-I.pdf)
34.  Case Control Study -II[Case Control Study II.pdf](https://github.com/atanustat/biostatonco/files/4001343/Case.Control.Study.II.pdf)
35. Confounding[confounding.pdf](https://github.com/atanustat/biostatonco/files/4001342/confounding.pdf)
36. Stratified Analysis[Stratified Analysis.pdf](https://github.com/atanustat/biostatonco/files/4001340/Stratified.Analysis.pdf)
37. Matched Analysis-I[Matched Analysis-I.pdf](https://github.com/atanustat/biostatonco/files/4001337/Matched.Analysis-I.pdf)
38.  Matched Analysis-I[Mathced Analysis-II.pdf](https://github.com/atanustat/biostatonco/files/4001336/Mathced.Analysis-II.pdf)



### Talk
1. Lung Cancer Update 2019 [presentation.pdf](https://github.com/atanustat/biostatonco/files/4002993/presentation.pdf).
2. Computational Biostatistics and Survival Analysis Talk[cce presentation.pdf](https://github.com/atanustat/biostatonco/files/4002995/cce.presentation.pdf).



### OpenBUGS code
 ### Model1:- Growth Curve Modeling 
  
   model {  
   
    for (i in 1:5) {
    
    y[i]       ~ dnorm(mu[i], tau)
    
    mu[i]     <- alpha + beta*(x[i] - mean(x[]))
    
    }
    
    alpha        ~ dflat()
    
    beta         ~ dflat()
    
    tau         <- 1/sigma2
    
    log(sigma2) <- 2*log.sigma
    
    log.sigma    ~ dflat()     
    
  }
  
  
  Data:
  
  list(y = c(177,236,285,350,376), x = c(8,15,22,29,36))
  
  Inits:
  
  list(alpha=250,beta=0,log.sigma=0)
  

### Model2:- Growth Curve Modeling 

model {

  for (i in 1:23) {
  
    y[i]             ~ dnorm(mu[i], tau)
    
    D2[i]           <- equals(DIST[i], 2)
    
    D3[i]           <- equals(DIST[i], 3)
    
    mu[i]           <- beta0 + beta[1]*MAN[i]
    
                     + beta[2]*D2[i] + beta[3]*D3[i]
                     
  }
  
  beta0              ~ dnorm(0, 0.0001)
  
  for (j in 1:3) {
  
    beta[j]          ~ dnorm(0, 0.0001)
    
  }
  
  tau               <- 1/pow(sigma, 2)
  
  sigma              ~ dunif(0, 100)
  

  dummy             <- AUTO[1]
  
}


Inits:

list(beta0 = 0, beta = c(0, 0, 0), sigma = 1)

Data:

MAN[]  AUTO[]  y[]  DIST[]

-15.76	1.09	3.19	1

0.98	0.62	-3.45	1

3.71	0.61	0.04	1

-5.37	-1.01	6.62	1

-10.23	-0.76	3.61	1

-8.32	1.91	2.67	1

-7.80	0.40	-2.45	1

6.77	-1.71	9.31	1

-8.81	-0.76	15.29	1

-9.56	-1.34	3.68	1

-2.06	-1.71	8.63	2

-0.76	-1.82	10.82	2

-6.30	-4.91	-0.50	2

39.40	-2.65	-11.00	2

-10.79	0.11	2.05	2

-8.16	0.52	11.80	2

-2.82	-2.54	-2.02	2

-16.19	-0.07	0.94	3

-11.00	-0.83	4.42	3

-14.60	0.98	-0.86	3

-17.96	-3.41	-0.92	3

0.76	2.97	2.61	3

-10.77	2.92	1.58	3

END



### Model 3:- New York Data Continued

model {

  for (i in 1:23) {
  
    y[i]             ~ dnorm(mu[i], tau)
    
    mu[i]           <- beta0 + beta[1]*MAN[i]
    
  }
  
  beta0              ~ dnorm(0, 0.0001)
  
  beta[1]            ~ dnorm(0, 0.0001)
  
  tau               <- 1/pow(sigma, 2)
  
  sigma              ~ dunif(0, 100)
  

  dummy             <- AUTO[1] + DIST[1]
  
}

Inits:

list(beta0 = 0, beta = c(0), sigma = 1)

### Model 4:- New York Data Continued

model {
  for (i in 1:23) {
    y[i]             ~ dt(mu[i], tau, dof)
    mu[i]           <- beta0 + beta[1]*MAN[i]
  }
  beta0              ~ dnorm(0, 0.0001)
  beta[1]            ~ dnorm(0, 0.0001)
  tau               <- 1/pow(sigma, 2)
  sigma              ~ dunif(0, 100)
  sd                <- sigma*sqrt(dof/(dof-2))

  dummy             <- AUTO[1] + DIST[1]
}

Inits:
list(beta0 = 0, beta = c(0), sigma = 1)

Data:
list(dof = 4)

### Model 5:-Dugongs
model {
   for(j in 1:N) {
     for (i in 1:4) {
       y[i,j]                ~ dnorm(mu[i,j], tau[i])
     }
     mu[1,j] <- Linf[1] - (Linf[1] - L0[1])*exp(-K[1]*x[1,j])
     mu[2,j] <- Linf[2] - (Linf[2] - L0[2])*exp(-K[2]*x[2,j])
     mu[3,j] <- alpha[3] - beta[3]*pow(gamma[3], x[3,j])
     mu[4,j] <- alpha[4] - beta[4]*pow(gamma[4], x[4,j])
   }
   L0[1]                     ~ dunif(0, 100)
   L0[2]                     ~ dnorm(0, 0.0001)I(0, Linf[2])
   Linf[1]                  <- L0[1] + beta[1]
   Linf[2]                   ~ dnorm(0, 0.0001)I(L0[2], )
   K[1]                      ~ dunif(0, 100)
   K[2]                      ~ dunif(0, 100)
   for (i in 1:2) {alpha[i] <- Linf[i]}
   for (i in 3:4) {alpha[i]  ~ dunif(0, 100)}
   beta[1]                   ~ dunif(0, 100)
   beta[2]                  <- Linf[2] - L0[2]
   for (i in 3:4) {beta[i]   ~ dunif(0, 100)}
   for (i in 1:2) {gamma[i] <- exp(-K[i])}
   gamma[3]                  ~ dunif(0, 1)
   gamma[4]                  ~ dgamma(0.001, 0.001)I(0, 1)
   for (i in 1:4) {
     tau[i]                 <- 1/sigma2[i]
     log(sigma2[i])         <- 2*log.sigma[i]
     log.sigma[i]            ~ dunif(-10, 10)
   }
 }

Inits:
list(alpha = c(NA, NA, 3, 3), beta = c(2, NA, 2, 2), gamma = c(NA, NA, 0.9, 0.9), K = c(0.1, 0.1), Linf = c(NA, 3), L0 = c(1, 1), log.sigma = c(-5, 0, -5, -5))

Data:
list(x = structure(
.Data = c(1.0,  1.5,  1.5,  1.5, 2.5,   4.0,  5.0,  5.0,  7.0,
	            8.0,  8.5,  9.0,  9.5, 9.5,  10.0, 12.0, 12.0, 13.0,
	           13.0, 14.5, 15.5, 15.5, 16.5, 17.0, 22.5, 29.0, 31.5,
				1.0,  1.5,  1.5,  1.5, 2.5,   4.0,  5.0,  5.0,  7.0,
	            8.0,  8.5,  9.0,  9.5, 9.5,  10.0, 12.0, 12.0, 13.0,
	           13.0, 14.5, 15.5, 15.5, 16.5, 17.0, 22.5, 29.0, 31.5,
				1.0,  1.5,  1.5,  1.5, 2.5,   4.0,  5.0,  5.0,  7.0,
	            8.0,  8.5,  9.0,  9.5, 9.5,  10.0, 12.0, 12.0, 13.0,
	           13.0, 14.5, 15.5, 15.5, 16.5, 17.0, 22.5, 29.0, 31.5,
				1.0,  1.5,  1.5,  1.5, 2.5,   4.0,  5.0,  5.0,  7.0,
	            8.0,  8.5,  9.0,  9.5, 9.5,  10.0, 12.0, 12.0, 13.0,
	           13.0, 14.5, 15.5, 15.5, 16.5, 17.0, 22.5, 29.0, 31.5),
.Dim = c(4, 27)),
y = structure(
.Data = c(1.80, 1.85, 1.87, 1.77, 2.02, 2.27, 2.15, 2.26, 2.47,
	           2.19, 2.26, 2.40, 2.39, 2.41, 2.50, 2.32, 2.32, 2.43,
	           2.47, 2.56, 2.65, 2.47, 2.64, 2.56, 2.70, 2.72, 2.57,
				1.80, 1.85, 1.87, 1.77, 2.02, 2.27, 2.15, 2.26, 2.47,
	           2.19, 2.26, 2.40, 2.39, 2.41, 2.50, 2.32, 2.32, 2.43,
	           2.47, 2.56, 2.65, 2.47, 2.64, 2.56, 2.70, 2.72, 2.57,
				1.80, 1.85, 1.87, 1.77, 2.02, 2.27, 2.15, 2.26, 2.47,
	           2.19, 2.26, 2.40, 2.39, 2.41, 2.50, 2.32, 2.32, 2.43,
	           2.47, 2.56, 2.65, 2.47, 2.64, 2.56, 2.70, 2.72, 2.57,
				1.80, 1.85, 1.87, 1.77, 2.02, 2.27, 2.15, 2.26, 2.47,
	           2.19, 2.26, 2.40, 2.39, 2.41, 2.50, 2.32, 2.32, 2.43,
	           2.47, 2.56, 2.65, 2.47, 2.64, 2.56, 2.70, 2.72, 2.57),
.Dim = c(4, 27)), N = 27)



### Model 6:-Jaws

  model {
    for (i in 1:20) {Y[i, 1:4] ~ dmnorm(mu[], Sigma.inv[,])}
    for (j in 1:4)  {mu[j]    <- alpha + beta*x[j]}
    alpha                      ~ dnorm(0, 0.0001)
    beta                       ~ dnorm(0, 0.0001)
    Sigma.inv[1:4, 1:4]        ~ dwish(R[,], 4)
    Sigma[1:4, 1:4]           <- inverse(Sigma.inv[,])
  }

  Inits:
	list(alpha = 40, beta = 1)

  Data:
  list(Y = structure(
           .Data = c(47.8, 48.8, 49.0, 49.7,
                     46.4, 47.3, 47.7, 48.4,
                     46.3, 46.8, 47.8, 48.5,
                     45.1, 45.3, 46.1, 47.2,
                     47.6, 48.5, 48.9, 49.3,
                     52.5, 53.2, 53.3, 53.7,
                     51.2, 53.0, 54.3, 54.5,
                     49.8, 50.0, 50.3, 52.7,
                     48.1, 50.8, 52.3, 54.4,
                     45.0, 47.0, 47.3, 48.3,
                     51.2, 51.4, 51.6, 51.9,
                     48.5, 49.2, 53.0, 55.5,
                     52.1, 52.8, 53.7, 55.0,
                     48.2, 48.9, 49.3, 49.8,
                     49.6, 50.4, 51.2, 51.8,
                     50.7, 51.7, 52.7, 53.3,
                     47.2, 47.7, 48.4, 49.5,
                     53.3, 54.6, 55.1, 55.3,
                     46.2, 47.5, 48.1, 48.4,
                     46.3, 47.6, 51.3, 51.8),
           .Dim =  c(20, 4)),
       x = c(8.0, 8.5, 9.0, 9.5),
       R = structure(
           .Data = c(4, 0, 0, 0,
                     0, 4, 0, 0,
                     0, 0, 4, 0,
                     0, 0, 0, 4),
           .Dim =  c(4, 4)))
           
  
### Model 7:-Beetles        
           
           model {
  for (i in 1:8) {
    y[i]         ~ dbin(p[i], n[i])
    logit(p[i]) <- alpha + beta*(x[i] - mean(x[]))
    phat[i]     <- y[i]/n[i]
    yhat[i]     <- n[i]*p[i]
  }
  alpha          ~ dnorm(0, 0.0001)
  beta           ~ dnorm(0, 0.0001)
}

Data:
list(x = c(1.6907, 1.7242, 1.7552, 1.7842, 1.8113, 1.8369, 1.8610, 1.8839),
     n = c(59, 60, 62, 56, 63, 59, 62, 60),
     y = c(6, 13, 18, 28, 52, 53, 61, 60))
	
Inits:
list(alpha = 50, beta = 0)

 
### Model 8:-Teatasting 
model {
  for (i in 1:2) { 
    y[i]         ~ dbin(p[i], n[i])

#  Independent priors 
#  a) standard uniform
    p[i]         ~ dunif(0,1)
#  b) Jeffreys
#   p[i]         ~ dbeta(0.5, 0.5)
#  c) uniform priors on the logit scale - note that the equivalent p[i] ~ dbeta(0,0) is not allowed in BUGS. 
#   logit(p[i]) <- alpha[i] 
#   alpha[i]     ~ dflat()
  }

# One-parameter priors 
# p[2]          <- 1 - p[1]
# a) uniform 
# One-parameter uninformative
# p[1]           ~ dunif(0, 1)
# b) Jeffreys
# p[1]           ~ dbeta(0.5, 0.5)
# c) logit-uniform
# logit(p[1])   <- alpha
# alpha          ~ dflat()

# One parameter, sceptical prior
# p[1]          <- theta[pick]
# pick           ~ dcat(q[])
# q[1]          <- 0.8
# q[2]          <- 0.2
# theta[1]      <- 0.5
# theta[2]       ~ dunif(0.5, 1)

  post          <- step(p[1] - p[2])

# Dependent priors
# a) uniform 
# p[1]           ~ dbeta(a, b)
# a             <- 1
# b             <- 1
# b) Jeffreys
# p[1]           ~ dbeta(a, b)
# a             <- 0.5
# b             <- 0.5
# c) logit-uniform
# a             <- 0
# b             <- 0
# logit(p[1])   <- alpha
# alpha          ~ dflat()

# with rho = 6/(1+1+6) = 0.75
# n.corr        <- 6
# or with rho = 14/(1+1+14) = 0.875
# n.corr        <- 14

# x              ~ dbin(p[1], n.corr)
# a.post        <- a + x
# b.post        <- b + n.corr - x
# p[2]           ~ dbeta(a.post, b.post)

# Altham's priors to mimic Fisher's exact test
# p[1]           ~ dbeta(0.00001, 1)
# p[2]           ~ dbeta(1,0.00001)

}

Data: base case where she gets six out of eight cups right. 
list(n=c(4,4), y=c(3,1))

Alternative scenario where she gets all of them right
list(n=c(4,4), y=c(4,0))

Initial values for independent logit-uniform priors
list(alpha=c(0,0))
Initial values for one-parameter logit-uniform priors
list(alpha = 0)
Initial values for independent probabilities (though gen.inits is sufficient)
list(p=c(0.5, 0.5))


### Contact
abhattacharjee@actrec.gov.in/atanustat@gmail.com   

Centre for Cancer Epidemiology, Tata Memorial Centre, Navi Mumbai,India
